doc/api/cluster.md

```diff
@@ -840,7 +855,8 @@ socket.on('data', (id) => {
   const worker = cluster.workers[id];
 });
 ```
-
+[`Worker`]: #cluster_class_worker
+[`Error`]: errors.html#errors_class_error
```

Comment:
It would be better if you place `Worker` and `Error` in alphabetical order.

Reply:
@oyyd Sure, I will update.